radicle-cob/src/history/entry.rs

```diff
@@ -11,7 +11,7 @@ use crate::pruning_fold;
 
 /// Entry contents.
 /// This is the change payload.
-pub type Contents = NonEmpty<Vec<u8>>;
+pub type Contents = NonEmpty<(Oid, Vec<u8>)>;
```

Comment:
I think it might be nice to make the `(Oid, Vec<u8>)` a `struct`. Field names would be nice for telling what it is and then also useful for using the `Entry` in the `traverse`, i.e. not using `.1` to get the inner contents.

Reply:
Yeah true.. maybe `EncodedOp` :thinking: 
Also need to change the comment, because that's actually only used on the "read" side, since the Oid isn't actually stored on disk.